index.js

```diff
@@ -46,24 +46,27 @@ exports.kinesisHandler = function (records, context, callback) {
           return parseKinesis(record, avroType)
         })
       // post to API
-      logger.info({'message': 'Posting records'})
 
-      postRecords(records, isBibOrTeacherSet(records))
+      updateCreateRecordsArray = []
+      deletedRecordsArray = []
+
+      records.forEach(function(record){
+        if(record.deleted){
```

Comment:
Where does the record.deleted property get filled?

Reply:
record.deleted get filled at Sierra if you delete the record or not. We're just checking to see if that value is true or false.